extensions/can-i-php/src/lib/types/preferences.ts

```diff
@@ -0,0 +1,8 @@
+export interface Preferences {
+  php_84: boolean;
+  php_83: boolean;
+  php_82: boolean;
+  php_81: boolean;
+  php_80: boolean;
+  php_74: boolean;
+}
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

Reply:
Good to know, thanks!